src/modules/users/components/Inbox/InboxIcon/InboxIcon.tsx

```diff
@@ -23,18 +18,8 @@ interface Props {
 
 const displayName = 'users.Inbox.InboxIcon';
 
-const Inbox = ({
-  activeClassName,
-  title = MSG.fallbackTitle,
-  hasUnreadActivities,
-}: Props) => (
-  <NavLink
-    to={INBOX_ROUTE}
-    className={hasUnreadActivities ? styles.inboxIconWCircle : styles.inboxIcon}
-    activeClassName={activeClassName}
-  >
-    <Icon name="envelope" title={title} />
-  </NavLink>
+const Inbox = ({ title = MSG.fallbackTitle }: Props) => (
```

Comment:
The variable name here should probably be `InboxIcon`; the display name is correct, though. We can also wait for this to be fixed in #1775